recipes/write_behind/README.md

```diff
@@ -0,0 +1,99 @@
+# RedisGears Write-Behind Recipe
+
+## System requirements
+
+* Redis v5.0.7 or above,
+* Relational database: MySQL (any version will do) or Oracle (tested with 11g and 12c) database,
+* Redis Enterprise Software v5.4.11-2 and above,
```

Comment:
This is a requirement only if you use RedisEnterprise

Reply:
I'll add the OSS version.